io.github.retgal.Dayon.yml

```
@@ -0,0 +1,51 @@
+app-id: io.github.retgal.Dayon
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: dayon_assistant.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --filesystem=home
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: dayon
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 dayon.jar -t /app/bin
+      - install -Dm755 dayon_*.sh -t /app/bin
+      - install -Dm755 dayon.flatpak.sh -t /app/bin/dayon.sh
+      - install -Dm755 dayon.browser -t /app/bin
+      - install -Dm644 dayon.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install -Dm644 dayon.assisted.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - desktop-file-edit --set-key=Exec --set-value='/app/bin/dayon_assisted.sh' /app/share/applications/${FLATPAK_ID}.desktop
+      - desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 dayon.assistant.desktop /app/share/applications/${FLATPAK_ID}.assistant.desktop
+      - desktop-file-edit --set-key=Exec --set-value='/app/bin/dayon_assistant.sh' /app/share/applications/${FLATPAK_ID}.assistant.desktop
+      - desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.assistant.desktop
```

I forgot. You need also to install the appdata file:

```suggestion
      - desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.assistant.desktop
      - install -Dm644 -t ${FLATPAK_DEST}/share/appdata io.github.retgal.Dayon.appdata.xml
```

---

Ok, that makes sense, thanks again!